### PR TITLE
Latest HAT version deployment into sandbox

### PR DESCRIPTION
**Pull request by augustinas**
The latest changes including:

- Scalafmt inclusion via sbt plugin
- Final fixes for the Portugese language support

### CODE REVIEW

**Comment by augustinas** [comment]
@dataswifty interestingly a lot of the changes in this PR are very similar to one of the PR https://github.com/Hub-of-all-Things/HAT2.0/pull/148 which is being merged directly into `staging` branch. Could you advise on the best way to proceed to get all of the changes merged into `staging`?

**Comment by dataswifty** [comment]
> @dataswifty interestingly a lot of the changes in this PR are very similar to one of the PR #148 which is being merged directly into `staging` branch. Could you advise on the best way to proceed to get all of the changes merged into `staging`?

This is the only diff: hat/conf/application.conf

File: .scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,29 @@
+include ".scalafmt-common.conf" 
+
+// ScalaFmt stuff
```
Comment (benshaw) [inline .scalafmt.conf:3]: Only the include is needed here everything else is in the common file

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,7 +1,16 @@
 logLevel := Level.Warn
 
+// Typesafe Resolvers
 resolvers += Resolver.typesafeRepo("releases")
 
+// S3 based SBT resolver
+resolvers ++= Seq(
+  "HAT Library Artifacts Releases" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com",
+  "HAT Library Artifacts Snapshots" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-snapshots.hubofallthings.com",
+  "HAT Library Artifacts Releases" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com"
```
Comment (benshaw) [inline project/plugins.sbt:10]: Releases appears twice

**Pull request merged**